x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -19,6 +19,12 @@ impl Address {
     }
 }
 
+impl Default for Address {
+    fn default() -> Self {
+        Address([0; 33])
```

Comment:
Nits... but it makes a difference if you ever go to make changes (you have to make fewer)
```suggestion
impl Default for Address {
    fn default() -> Self {
        Self([0; Self::LEN])
```
I was going to say you can just derive `Default`... but we're one element too long:
https://doc.rust-lang.org/stable/std/default/trait.Default.html#impl-Default-for-%5BT;+32%5D